doc/source/admin/jobs.md

```diff
@@ -85,6 +85,14 @@ exec
 raw
 : Disable auto-quoting of values when setting up environment variables.
 
+For containerized execution the environment setup done by ``file`` and ``exec`` ``env`` tags will
+not be available in the container, but only to the pre-and-post-tool-execution job environment.
+Instead, for containerized destinations variables that should only be available in the container
+can be set with ``<param id="docker_env_VARIABLE">VALUE</param>`` and
+``<param id="singularity_env_VARIABLE">VALUE</param>``, respectively.
```

Comment:
Maybe if you mention this is a known bug - it would help alleviate @mvdbeek's concerns below somewhat?